runtime/src/lib.rs

```diff
@@ -26,6 +24,14 @@ use sp_runtime::{
 	ApplyExtrinsicResult,
 };
 
+#[cfg(test)]
+mod mock;
+#[cfg(test)]
+mod tests;
```

Comment:
...what about a common file in runtime? Even though it is our own special implementation different from upstream, it will be shared by our future kusama/polkadot parachain.